app/extension/maven-plugin/pom.xml

```diff
@@ -368,6 +379,9 @@
       <plugin>
         <artifactId>maven-dependency-plugin</artifactId>
         <configuration>
+          <ignoredUsedUndeclaredDependencies>
+            <ignoredUsedUndeclaredDependency>junit:junit</ignoredUsedUndeclaredDependency>
+          </ignoredUsedUndeclaredDependencies>
```

Comment:
Is junit unused? Doesn't seem so, perhaps it's in wrong scope?

Reply:
exclude it from org.codehaus.plexus:plexus-container-default or set junit:junit's scope to provided perhaps

Reply:
fixed